.gitignore

```diff
@@ -201,3 +201,4 @@ src/Umbraco.Tests/TEMP/
 
 /src/Umbraco.Web.UI/config/umbracoSettings.config
 /src/Umbraco.Web.UI.NetCore/Umbraco/models/*
+/src/Umbraco.Web.UI.NetCore/appsettings.local.json

```

Comment:
@bergmania Does the .gitignore file also get pushed into the template, or is this change something that developers would need to know to add?

Reply:
Currently not